docs/docusaurus/docs/cloud/about_gx.md

```diff
@@ -30,47 +32,38 @@ The following diagram provides an overview of the key GX Cloud architecture comp
 
 - **GX Cloud web UI** - A web interface that allows you to manage and validate your organization's data quality without running Python code and enables shared visibility into your organization's Validation Results and Checkpoint run history. It's browser- and platform-independent.
 
-- **GX Cloud API** - Provides a REST API to programmatically access and manage GX Cloud data and configurations. Both the GX open source software (OSS) client and the GX Agent use the GX Cloud API to query data from and send data to GX Cloud. Documentation for the GX Cloud API is not currently available, and it is not yet intended for use outside the GX OSS client and the GX Agent.
-
-- **GX Cloud message broker** - Enables communication between GX Cloud and the GX Agent.
-
-### Deployment components
-
-- **GX OSS client** - The Python library that powers GX Cloud and provides a Python client for programmatic access to GX Cloud. GX OSS contains the logic needed to test and document your organization's data, and you can also use it to create, manage, and interact with GX Cloud components.
```

Comment:
If the GX OSS Client is in the Cloud diagram, we should retain its description in the accompanying text. Otherwise, it introduces confusion for the reader.

GX Cloud (web interface) and GX Core (Python interface) are complementary parts of the complete GX Platform, and I think we could do a clearer job on this page of explaining that to the reader.

(I have a related renaming suggestion on the OSS/Core client on the diagram file).

Reply:
Where do you think it should go? I didn't add it to the GX Cloud components section because it's not really part of GX Cloud.